ios-training/Model/WeatherModel.swift

```diff
@@ -11,10 +11,32 @@ import YumemiWeather
 class WeatherModel {
     weak var delegate: WeatherDelegate? = nil
     
+    let df: DateFormatter = {
+        let df = DateFormatter()
+        df.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZZ"
+        return df
+    }()
+    
+    func jsonEncode(from request: Request) throws -> String {
```

Comment:
このあたりもprivateでよさそうです。
private にしたメソッド、変数はpublicのものより下に配置することが多いです。

```suggestion
    func jsonEncode(from request: Request) throws -> String {
```